projects/tinyobjloader/patch.diff

```diff
@@ -0,0 +1,48 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Please uploaded patches for these in upstream repo, we cant land diffs here.

Reply:
https://github.com/tinyobjloader/tinyobjloader/pull/302

Reply:
@inferno-chromium it was merged, and the patch.diff is removed